src/olympia/versions/models.py

```diff
@@ -796,14 +826,21 @@ def disable_old_files(self):
                 f.update(status=amo.STATUS_DISABLED)
 
     def reset_due_date(self, due_date=None):
```

Comment:
Only use of the parameter - which is confusing me - seems to be in tests ?

Reply:
it's used by `inherit_due_date` just below

Reply:
Ah, I had missed that because it was part of the previous patch, and it was simpler back then... I would love a docstring on here as well to make what it's doing clearer

Reply:
add some docstrings